test/Microsoft.Extensions.CommandLineUtils.Tests/DotNetMuxerTests.cs

```diff
@@ -15,6 +15,7 @@ public void FindsTheMuxer()
             var muxerPath = DotNetMuxer.MuxerPath;
             Assert.NotNull(muxerPath);
             Assert.True(File.Exists(muxerPath), "The file did not exist");
+            Assert.True(Path.IsPathRooted(muxerPath), "The path should be rooted");
```

Comment:
Could you also add a Assertion that says the file name is `dotnet.exe`? Just so we know in case they ever break us.